manifests/init.pp

```diff
@@ -990,7 +990,7 @@
             require => Package[$my_package_name],
           }
 
-          file { 'pam_common_noninteractive_session':
+          file { 'pam_common_session_noninteractive':
```

Comment:
Why change the resource title? This could potentially be construed as a breaking change if someone tightly coupled their module to ours and they have a relationship to this resource.